pylint/config.py

```diff
@@ -162,6 +162,9 @@ def _regexp_validator(_, name, value):
         return value
     return re.compile(value)
 
+# pylint: disable=unused-argument
```

Comment:
What argument is not used? It seems that both name and value are used and the first one is marked as unused anyway with underscore.